### PR TITLE
chore: prepare docs and other chores for v10

### PR DESCRIPTION
**Pull request by balzss**

### CODE REVIEW

File: docs/getting-started/usage.md [hunk 1]
```diff
@@ -14,31 +14,34 @@ The following steps will create a React app that uses Instructure UI. Recommende
...
+}
 }
 ```
 
 Run `npm install`, so InstUI is downloaded to your `node_modules` folder and can be used.
 
-##### Try the sample code 
+##### Try the sample code
+
 Now you are ready to use InstUI, let's try it out. Replace the code in `App.js` with the following:
```
Comment (matyasf) [inline docs/getting-started/usage.md:45]: later in this file it talks about the `instructure` theme, can you remove that part?

**Review by matyasf — CHANGES_REQUESTED** [review]
in `CodeSandboxButton` the dependencies need to be updated to v10

File: docs/getting-started/usage.md (hunk 1 above)
Comment (balzss) [reply docs/getting-started/usage.md:45]: done

**Comment by balzss** [comment]
> in `CodeSandboxButton` the dependencies need to be updated to v10

done

**Review by git-nandor — APPROVED** [review]

**Review by matyasf — APPROVED** [review]

**Pull request merged**